src/client/mod.rs

```diff
@@ -82,47 +82,30 @@ fn read_config(p: &Path) -> SshConfig {
 
 fn connect(host: &str, params: &HostParams) {
     // Resolve host
-    let host = match params.host_name.as_deref() {
-        Some(h) => h,
-        None => host,
-    };
-    let port = match params.port {
-        None => 22,
-        Some(p) => p,
-    };
-    let host = match host.contains(':') {
-        true => host.to_string(),
-        false => format!("{}:{}", host, port),
+    let host = params.host_name.as_deref().unwrap_or(host);
+    let port = params.port.unwrap_or(22);
+
+    let formatted_host : String;
+    let host: &str = if host.contains(':') {
```

Comment:
is this backwards? Don't you want if !host.contains?

Reply:
Yea this is Rust's notion of a ternary expression, a little back-and-forth reading
sorta `host.contains(':') ? host : ...` in other languages

Reply:
oh nvm for a second I thought it was a split but it's not. Got it.